conjure-java-core/src/test/resources/test/api/TestServiceBlocking.java.dialogue.prefix

```diff
@@ -60,33 +60,33 @@ public interface TestServiceBlocking {
      * @apiNote {@code POST /catalog/datasets}
      */
     @ClientEndpoint(method = "POST", path = "/catalog/datasets")
-    Dataset createDataset(AuthHeader authHeader, String testHeaderArg, CreateDatasetRequest request);
+    Dataset createDataset(AuthHeader authHeader, @Safe String testHeaderArg, CreateDatasetRequest request);
```

Comment:
We previously didn't include safety from markers, only tags! I don't think that was intentional